packages/compass-e2e-tests/helpers/selectors.ts

```diff
@@ -1400,3 +1400,21 @@ export const ProxyCustomButton =
 
 // Close tab confirmation
 export const ConfirmTabCloseModal = '[data-testid="confirm-tab-close"]';
+
+export const GlobalWrites = {
+  tabStatus: (status: string) => `[data-status="${status.toLowerCase()}"]`,
```

Comment:
Small nit: I think you might want to follow @paula-stacho advice here and add both data-testid as a special test selector in addition to the data-status, otherwise the selector you're building is a bit too vague about the context of selection (status of what?) which might make it hard to debug